tests/api.c

```diff
@@ -46165,7 +46165,7 @@ static int test_wolfSSL_EC_KEY_print_fp(void)
 {
     int res = TEST_SKIPPED;
 #if defined(HAVE_ECC) && ((defined(HAVE_ECC224) && defined(HAVE_ECC256)) || \
-    defined(HAVE_ALL_CURVES)) && ECC_MIN_KEY_SZ <= 256 && \
+    defined(HAVE_ALL_CURVES)) && ECC_MIN_KEY_SZ <= 224 && \
```

Comment:
I don't understand why this was changed `ECC_MIN_KEY_SZ <= 224`? Should it be `ECC_MIN_KEY_SZ >= 224`?

Reply:
Nope.  must be less than or equal to 224 because we are also using `NID_X9_62_prime256v1` in this test.